doc/howto/multifederation.md

```diff
@@ -6,82 +6,93 @@ This document provides an initial step by step description of the technical solu
 
 The description of the steps is based on the diagram that Steffen draw in the board.
 
+
 ## Entities
-* `RP`. A Relying Party.
-* `OP`. A OpenID Provider.
-* `SS_FA`. The Signing Service for Federation A (`FA`).
-* `SS_FB`. The Signing Service for Federation B (`FB`).
-* `SS_FE`. The Signing Service for the EduGAIN federation (`FE`).
+* `RP1`. A Relying Party.
+* `OP1`. A OpenID Provider.
+* `MSS_FA`. The Metadata Signing Service for Federation A (`FA`).
+* `MSS_FB`. The Metadata Signing Service for Federation B (`FB`).
+* `MSS_CE`. The Metadata Signing Service for the community of federations E (`CE`). This might be eduGAIN-OIDC..
```

Comment:
Remove the double point

Reply:
Will do.

Reply:
The moment you upload the commit I will merge